planning/behavior_path_planner/src/scene_module/lane_change/normal.cpp

```diff
@@ -861,9 +861,11 @@ bool NormalLaneChange::hasEnoughLengthToCrosswalk(
   const auto & route_handler = *getRouteHandler();
   const auto overall_graphs_ptr = route_handler.getOverallGraphPtr();
 
+  const double base_link2front = planner_data_->parameters.base_link2front;
   const double dist_to_crosswalk_from_lane_change_start_pose =
     utils::getDistanceToCrosswalk(current_pose, current_lanes, *overall_graphs_ptr) -
-    path.info.length.prepare;
+    path.info.length.prepare - base_link2front;
```

Comment:
It seems okay.
Could you add same changes `in hasEnoughLengthToIntersection()`?

https://github.com/autowarefoundation/autoware.universe/blob/9ca718d88983b44f6619e361bf43c050c9d6d660/planning/behavior_path_planner/src/scene_module/lane_change/normal.cpp#L886-L887